skins/cat17/src/sass/layouts/pages/donation_confirmation.scss

```diff
@@ -128,9 +128,42 @@
 	}
 
 	.membership-benefits-box {
+		h2 {
+			border-bottom: solid 2px $brand-primary-membership;
+			display: inline-block;
+
+			/* Counteract styles from _layouts/memberships */
+			@include bkp(xsmax) {
+				&::after {
+					border-bottom:none;
```

Comment:
Ultra nitpick, but there should be a space after the colon character, doesn't our code style toolset check for this?